charts/datadog/templates/gke_autopilot_allowlist_synchronizer.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and .Values.providers.gke.autopilot (not (eq .Values.datadog.envDict.DD_CI true))}}
```

Comment:
`(not (eq .Values.datadog.envDict.DD_CI true))` is a workaround to skip applying the `AllowlistSynchronizer` in non-Autopilot k8s clusters in CI.